packages/tools/client-debugger/client-debugger-view/src/index.ts

```diff
@@ -4,20 +4,20 @@
  */
 
 /**
- * Contains an extensible debug visualizer / editor for the Fluid client.
+ * This library contains a simple React-based visualizer for the Fluid Devtools.
  *
  * @remarks
  *
- * The entry-point to this package is {@link RootView}, a {@link https://react.dev/reference/react/Component | React Component}
+ * The entry-point to this package is {@link DevtoolsPanel}, a {@link https://react.dev/reference/react/Component | React Component}
```

Comment:
This is an unfortunate requirement for API-Extractor. Motivation for us to clean up internal naming in the future (or now, if we would prefer).

Reply:
Yup. Definitely seems like they should be able to resolve aliases, but alas.